src/main/Panels/ContentPanel.java

```diff
@@ -14,25 +14,25 @@ public class ContentPanel extends JPanel {
     private JButton viewButton;
     private GridBagConstraints constraints = new GridBagConstraints();
     private int id;
-    Database database = new Database();
+    public Database database = new Database();

```

Comment:
this is inconsistency since we pass db instance from Application to all panels but here we create another connection, which couses runtime Exception as database.close() is not called and we have limited amount of avaliable connetions.